oneoffs/oneoff_utils.py

```diff
@@ -1,75 +1,105 @@
+
+    positions, moves = zip(*[(p.position, p.next_move)
+                             for p in replay_sgf(sgf_contents)])
+    return positions, moves, result, props
 
-  positions, moves = zip(*[(p.position, p.next_move) for p in replay_sgf(sgf_contents)])
-  return positions, moves, result, props
 
 def check_year(props, year):
-  if year is None:
-    return True
-  if props.get('DT') is None:
-    return False
+    if year is None:
+        return True
+    if props.get('DT') is None:
+        return False
 
-  try:
-    #Most sgf files in this database have dates of the form
-    #"2005-01-15", but there are some rare exceptions like
-    #"Broadcasted on 2005-01-15.
-    year_sgf = int(props.get('DT')[0][:4])
-  except:
-    return False
-  return year_sgf >= year
+    try:
+        # Most sgf files in this database have dates of the form
+        #"2005-01-15", but there are some rare exceptions like
+        #"Broadcasted on 2005-01-15.
+        year_sgf = int(props.get('DT')[0][:4])
+    except:
+        return False
+    return year_sgf >= year
 
 
 def check_komi(props, komi_str):
-  if komi_str is None:
-    return True
-  if props.get('KM') is None:
-    return False
-  return props.get('KM')[0] == komi_str
-
-
-def find_and_filter_sgf_files(base_dir, min_year = None, komi = None):
-  sgf_files = []
-  count = 0
-  print("Finding all sgf files in {} with year >= {} and komi = {}".format(base_dir, min_year, komi))
-  for i, (dirpath, dirnames, filenames) in tqdm(enumerate(os.walk(base_dir))):
-    for filename in filenames:
-      count+=1
-      if count%5000 == 0:
-        print("Parsed {}, Found {}".format(count, len(sgf_files)))
-      if filename.endswith('.sgf'):
-        path = os.path.join(dirpath, filename)
-        props = get_sgf_props(path)
-        if check_year(props, min_year) and check_komi(props, komi):
-          sgf_files.append(path)
-  print("Found {} sgf files matching filters".format(len(sgf_files)))
-  return sgf_files
+    if komi_str is None:
+        return True
+    if props.get('KM') is None:
+        return False
+    return props.get('KM')[0] == komi_str
+
+
+def find_and_filter_sgf_files(base_dir, min_year=None, komi=None):
+    sgf_files = []
+    print("Finding all sgf files in {} with year >= {} and komi = {}".format(
+        base_dir, min_year, komi))
+    count = 0
+    for dirpath, dirnames, filenames in tqdm(os.walk(base_dir)):
+        for filename in filenames:
+            count += 1
+            if count % 5000 == 0:
+                print("Parsed {}, Found {}".format(count, len(sgf_files)))
+            if filename.endswith('.sgf'):
+                path = os.path.join(dirpath, filename)
+                props = get_sgf_props(path)
+                if check_year(props, min_year) and check_komi(props, komi):
+                    sgf_files.append(path)
+    print("Found {} sgf files matching filters".format(len(sgf_files)))
+    return sgf_files
+
+
+def get_model_paths(model_dir):
```

Comment:
should this and the rl_loop utils around model wrangling move to a common file?

Reply:
Probably but that's for a follow up. These were already present in training_curve and I'd like to submit this PR with as few modifications to the base directory as possible.